ts/navigation/TabNavigator.tsx

```diff
@@ -109,7 +108,7 @@ export const MainTabNavigator = () => {
             )
           }}
         />
-        {showBarcodeScanSection && (
+        {isSettingsVisibleAndHideProfile && (
```

Comment:
i think it's better use `isNewScanSectionLocallyEnabled` (see the next comments to understand why)
```
const isNewScanSectionLocallyEnabled = useIOSelector(
    isNewScanSectionLocallyEnabledSelector
  );
```

Reply:
Same as above, since the remote feature flag is still required for that section, It should be used this new one